pkg/server/events.go

```diff
@@ -93,16 +138,12 @@ func (em *eventMonitor) start() (<-chan struct{}, error) {
 // stop stops the event monitor. It will close the event channel.
 // Once event monitor is stopped, it can't be started.
 func (em *eventMonitor) stop() {
+	em.backOff.stopBackOff()
 	em.cancel()
 }
 
 // handleEvent handles a containerd event.
-func (em *eventMonitor) handleEvent(evt *events.Envelope) {
-	any, err := typeurl.UnmarshalAny(evt.Event)
-	if err != nil {
-		logrus.WithError(err).Errorf("Failed to convert event envelope %+v", evt)
-		return
-	}
+func (em *eventMonitor) handleEvent(any interface{}) error {
```

Comment:
Let's return error with description from this function, and we only need to log the error in `start`.

In this way, we don't need so many logs in this function.